.github/workflows/manylinux/entrypoint.sh

```diff
@@ -41,7 +44,7 @@ export CMAKE_ARGS="-DONNX_USE_LITE_PROTO=ON"
 $PIP_INSTALL_COMMAND -r requirements-release.txt || { echo "Installing Python requirements failed."; exit 1; }
 
 # Build wheels
-if [ "$GITHUB_EVENT_NAME" == "schedule" ] || [ "$GITHUB_EVENT_NAME" == "workflow_dispatch" ]; then
+if [ "$BUILD_MODE" == "preview" ]; then
```

Comment:
I would compare with release. This way if anything goes wrong and build mode is not set properly the default is preview.